### PR TITLE
Client side support for scouting protection

### PR DESCRIPTION
**Pull request by xfix**
Needed for Zarel/Pokemon-Showdown#1968.

### CODE REVIEW

**Comment by Zarel** [comment]
Seems a bit verbose.

**Comment by Zarel** [comment]
Consider

> Scouting is now allowed (Tournament players can watch other tournament battles)

and

> Scouting is now banned (Tournament players can't watch other tournament battles)

**Comment by xfix** [comment]
Bump.

File: js/client-chat-tournament.js [hunk 1]
```diff
@@ -332,6 +332,15 @@
 					}
 					break;
 
+				case 'scouting':
+					if (data[0] === 'allow') {
+						this.room.$chat.append("<div class=\"notice tournament-message-scouting\">Scouting is now allowed (Tournament players can watch other tournament battles)</div>");
+					}
+					else if (data[0] === 'disallow') {
```
Comment (Zarel) [inline js/client-chat-tournament.js]: Style.

`} else if` etc

Comment (xfix) [inline js/client-chat-tournament.js]: @Zarel I keep forgetting about it, considering it's one of three (or so) differences between my own style and Showdown style.

**Comment by xfix** [comment]
Bump, I guess.

**Pull request merged**